libs/shared/src/types/subscriber/index.ts

```diff
@@ -19,6 +19,16 @@ export interface ISubscribersDefine extends ISubscriberPayload {
   subscriberId: string;
 }
 
+export interface ISubscribersSource extends ISubscribersDefine {
+  _subscriberSource: SubscriberSourceEnum;
+}
+
+export enum SubscriberSourceEnum {
+  BROADCAST = 'broadcast',
+  SINGLE = 'single',
```

Comment:
Was not sure about the naming here, not on the enum and not on the `single`.

Reply:
I think this is good

Reply:
I agree the naming makes sense, it aligns with Rate Limting nomenclature.